src/map/entities/battleentity.h

```diff
@@ -143,7 +143,8 @@ enum SKILLTYPE
 	SKILL_ALC			= 55,
 	SKILL_COK			= 56,
 	SKILL_SYN			= 57,
-	SKILL_RID			= 58
+	SKILL_RID			= 58,
+	SKILL_DIG			= 59 // Chocobo digging

```

Comment:
I'm not 100% positive that it's NOT the same, but demo looked it up in polutils earlier and said it was 59.